mozci/push.py

```diff
@@ -1037,123 +1036,3 @@ def make_push_objects(**kwargs):
-
-    When invoked with a `type` argument, this method will return a list of
-    Summary objects of the corresponding type. For example, `type='group'` will
-    return a list of GroupSummary objects.
-
-    Args:
-        from_date (str): String representing an acceptable date value in ActiveData.
-        to_date (str): String representing an acceptable date value in ActiveData.
-        branch (str): String that references one of the Mozilla CI's repositories.
-        type (str): String that references one of the supported Summary types.
-
-    Returns:
-        list: List of Summary objects, or an empty list.
-
-    """
-    # Retrieve the function by name using the provided `type` argument.
-    func = "__make_{}_summary_objects".format(type).lower()
-
-    # If the method name `func` (maps to either of the private methods)
-    # is not found, then return an empty list.
-    if not func:
-        return []
-
-    # Obtain list of all pushes for the specified branch, from_date and to_date.
-    pushes = make_push_objects(from_date=from_date, to_date=to_date, branch=branch)
-
-    summaries = globals()[func](pushes, branch)
-
-    # Sort by either the label (LabelSummary) or name (GroupSummary).
-    summaries.sort(key=lambda x: (getattr(x, "label", "name")))
-    return summaries
-
-
-def __make_label_summary_objects(pushes, branch):
-    """Generates a list of LabelSummary objects from a list of pushes.
-
-    Args:
-        pushes (list): List of Push objects.
-
-    Returns:
-        list: List of LabelSummary objects.
-    """
-    # Flatten list of tasks for every push to a 1-dimensional list.
-    tasks = sorted(
-        [task for push in pushes for task in push.tasks], key=lambda x: x.label
-    )
-
-    # Make a mapping keyed by task.label containing list of Task objects.
-    tasks_by_config = defaultdict(lambda: defaultdict(list))
-    for task in tasks:
-        tasks_by_config[task.configuration][task.label]["tasks"].append(task)
-
-    return [
-        LabelSummary(label, tasks)
-        for config in tasks_by_config.keys()
-        for label, tasks in tasks_by_config[config].items()
-    ]
-
-
-def __make_group_summary_objects(pushes, branch):
-    """Generates a list of GroupSummary objects from a list of pushes.
-
-    Args:
-        pushes (list): List of Push objects.
-
-    Returns:
-        list: List of GroupSummary objects.
-    """
-    import adr
-
-    # Obtain all the task.id values contained in the pushes. It will be used in
-    # the `where` query against ActiveData.
-    results = []
-    revs = [p.rev for p in pushes]
-    # if we have too many revisions, ActiveData returns an error
-    for i in range(0, len(revs), 30):
-        for revs_chunk in revs[i : i + 30]:
-            try:
-                results += adr.query.run_query(
-                    "group_durations", Namespace(push_ids=revs_chunk, branch=branch)
-                )["data"]
-            except adr.MissingDataError:
-                pass
```

Comment:
Oh, I just noticed this has a fallback. @marco-c  is this still being used?

Reply:
The plan was to use this to show durations instead of # of groups for shadow scheduler evaluation, but we never ended up using it.
https://github.com/mozilla/mozci/issues/359 is tracking rewriting this without AD, but for the time being we can safely remove it.